specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/stable/2023-03-15/migrate.json

```diff
@@ -5153,7 +5153,85 @@
     },
     "AssessedDisk": {
       "type": "object",
-      "description": "Assessed Disk.",
+      "properties": {
+        "suitability": {
+          "$ref": "#/definitions/CloudSuitability",
+          "description": "Gets the suitability."
+        },
+        "suitabilityExplanation": {
+          "$ref": "#/definitions/AzureDiskSuitabilityExplanation",
+          "description": "Gets the suitability explanation."
+        },
+        "suitabilityDetail": {
+          "$ref": "#/definitions/AzureDiskSuitabilityDetail",
+          "description": "Gets the suitability detail."
+        },
+        "recommendedDiskSize": {
+          "$ref": "#/definitions/AzureDiskSize",
+          "description": "Gets the recommended disk size."
+        },
+        "recommendedDiskType": {
+          "$ref": "#/definitions/AzureDiskType",
+          "description": "Gets the recommended disk type."
+        },
+        "gigabytesForRecommendedDiskSize": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets the recommended disk size."
+        },
+        "recommendDiskThroughputInMbps": {
```

Comment:
>recommendDiskThroughputInMbps [](http://example.com/codeflow?start=9&length=29)

recommendDiskThroughputInMbps or recommendedDiskThroughputInMbps?

Reply:
its recommendDiskThroughputInMbps in the service implementation. we've had this property in our api contract for a while now and while its better to name it recommendedDiskThroughputInMbps, we wouldnt want to take this breaking change now